src/DataSource/Http.hs

```diff
@@ -130,35 +153,93 @@ listUnassignedTickets = do
 
     iteratePages req
 
-listAdminAgents :: forall m. (MonadIO m, MonadReader Config m) => m [User]
+listAdminAgents
+    :: forall m. (MonadIO m, MonadReader Config m)
+    => m [User]
 listAdminAgents = do
     cfg <- ask
     let url = showURL AgentGroupURL
     let req = apiRequest cfg url
 
     iteratePages req
 
+-- | Export tickets from Zendesk - https://developer.zendesk.com/rest_api/docs/core/incremental_export
+-- NOTE: If count is less than 1000, then stop paginating.
+-- Otherwise, use the next_page URL to get the next page of results.
+getExportedTickets
+    :: forall m. (MonadIO m, MonadReader Config m)
+    => ExportFromTime
+    -> m [TicketInfo]
+getExportedTickets time = do
```

Comment:
This function seems to be printing a lot of unnecessary printing.. Can you clean it up?

Reply:
I wouldn't say it's "unnecessary" since I required that in order to run it and find three major issues.
I have an alternative though, will do it in a separate ticket.

Reply:
Thanks!